Command/ArchiveCommand.php

```diff
@@ -39,6 +40,24 @@ protected function configure()
         ;
     }
 
+    /**
+     * If pdo data is set we use it, if not doctrine is.
+     */
+    private function getConnection()
+    {
+        if ($this->getContainer()->getParameter('acilia_dblogger')) {
+            $config =  $this->getContainer()->getParameter('acilia_dblogger');
+            if ($this->connection = null) {
```

Comment:
Ojo acá, estás **asignando** `null`, no comparando!
Recomiendo usar identidad `$this->connection === null`